src/routes/jira/configuration/jira-configuration-get.ts

```diff
@@ -113,20 +112,60 @@ export const JiraConfigurationGet = async (
 				isGlobalInstall: installation.repository_selection === "all"
 			}));
 
+		/**
+		 * TODO: fetch the actual list of GHE servers
+		 *
+		 * The following array is a sample
+		 * [
+		 * 		{
+		 * 			id: 1,
+		 * 			url: "https:/github.internal.company.com",
+		 * 			applications: [
+		 * 				{
+		 * 					id: 1,
+		 * 					name: "ghe-app-for-jira",
+		 * 					successfulConnections: [1, 2, 3].map(() => successfulConnections[0])
+		 * 				},
+		 * 				{
+		 * 					id: 2,
+		 * 					name: "inter-team-app",
+		 * 					successfulConnections: [1, 2, 3, 4].map(() => successfulConnections[0])
+		 * 				}
+		 * 			]
+		 * 		},
+		 * 		{
+		 * 			id: 2,
+		 * 			url: "https:/github.internal.oooohanothercompany.com",
+		 * 			applications: [
+		 * 				{
+		 * 					id: 1,
+		 * 					name: "apps-galore",
+		 * 					successfulConnections: [1, 2, 3, 4, 5].map(() => successfulConnections[0])
+		 * 				},
+		 * 				{
+		 * 					id: 2,
+		 * 					name: "loose-canon",
+		 * 					successfulConnections: [1, 2, 3, 4].map(() => successfulConnections[0])
+		 * 				}
+		 * 			]
+		 * 		}
+		 * ]
+ 		 */
+		const gheServers = [];
+
 		const handleNavigationClassName = await booleanFlag(BooleanFlags.GHE_SERVER, false, jiraHost)
 			? "select-github-version-link"
 			: "add-organization-link";
 
 		res.render("jira-configuration.hbs", {
 			host: jiraHost,
-			successfulConnections,
-			failedConnections,
-			hasConnections: !!installations.total,
+			gheServers,
+			gheCloud: { successfulConnections, failedConnections },
```

Comment:
The `e` in `gheCloud`, does it refer to `enterprise`? Is it intended or should it be `ghCloud`?

Reply:
Nice catch! Renamed to `ghCloud`.